src/Uno.UI/UI/Xaml/Controls/ContentDialog/ContentDialog.cs

```diff
@@ -282,9 +289,12 @@ void Complete(ContentDialogButtonClickEventArgs a)
 				if (!a.Cancel)
 				{
 					const ContentDialogResult result = ContentDialogResult.Secondary;
-					_tcs.SetResult(result);
 					SecondaryButtonCommand.ExecuteIfPossible(SecondaryButtonCommandParameter);
-					Hide(result);
+					if (Hide(result))
+					{
+						Console.WriteLine("tcs secondary");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5229032)